statut/statut.md

```diff
@@ -298,21 +318,24 @@ Na fundusze Stowarzyszenia składają się:
 5) wpływy z loterii i aukcji,
 6) dotacje budżetowe, subwencje,
 7) dochody z majątku ruchomego i nieruchomego,
-8) inne wpływy.
+8) dotacje,
+9) inne wpływy.
 
-#### §41
+#### §42
 
-Dla ważności dokumentów dotyczących zobowiązań finansowych wymagane są podpisy Prezesa i Skarbnika, względnie dwóch innych uprawnionych członków Zarządu.
+Dla ważności dokumentów dotyczących zobowiązań finansowych wymagany jest podpis:
+1) Prezesa i Skarbnika, względnie dwóch innych uprawnionych członków Zarządu, albo
```

Comment:
Nie jestem przekonany czy ta forma zapisu (wypunkowanie które jest alternatywą, bez informacji że jest to alternatywa przed wypunktowaniem) jest zrozumiała. Ja chyba bym zrobił to bez wypunktowania.